ledger/db/src/lib.rs

```diff
@@ -45,10 +47,11 @@ use std::{
 pub use error::Error;
 pub use ledger_trait::{Ledger, MockLedger};
 pub use mc_util_lmdb::{MetadataStore, MetadataStoreError};
+pub use mint_config_store::{ActiveMintConfig, MintConfigStore};
 pub use tx_out_store::TxOutStore;
 
 pub const MAX_LMDB_FILE_SIZE: usize = 2usize.pow(40); // 1 TB
-pub const MAX_LMDB_DATABASES: u32 = 22; // maximum number of databases in the lmdb file
+pub const MAX_LMDB_DATABASES: u32 = 24; // maximum number of databases in the lmdb file
```

Comment:
Maybe it would help to note somewhere in the PR body that we are adding 2 DBs? A mapping of token id -> currently active mint configurations and a mapping of nonce -> SetMintConfigTx?

Reply:
I think that becomes clear when reading the code, not sure where I would note this / who would ever look at it after it gets merged.